### ISSUE TITLE
AdsAbs running out of usages

### ISSUE THREAD

**Issue by GlazerMann** [ms609/citation-bot#1730]
can we get the number increased?

**Comment by ms609** [comment]
It would be good to have an indication of how much additional bandwidth we require. How quickly (how soon in each day?) are we hitting our limits?

**Comment by GlazerMann** [comment]
Seems it has reset for today, lets see how fast its used up this time. 1/25000 was used between 19:52 and 19:55 according to web page log on my current run. Redalert2fan (talk) 20:19, 7 June 2019 (UTC)
1 hour later and past 7000 searches. Redalert2fan (talk) 20:59, 7 June 2019 (UTC)
After 4 hours we are just above 22000 searches, so we have nearly run out again. Rate has slightly gone down since the reset but that seems to be due to only me using the bot. While usage may be higher than before due to the unblock, and me currently running the list that I have basically saved up probably does not help, it is not out of the question that multiple users on a "normal" day could also use up all of the searches. --Redalert2fan (talk) 00:05, 8 June 2019 (UTC)

**Comment by ms609** [comment]
Looks like we need to be requesting at least an order of magnitude increase then.  I'll feed this on to the AdsAbs maintainers and see what they'll be able to offer us.

**Comment by ms609** [comment]
AdsAbs have suggested a willingness to double our current limit, but only if this is still necessary once we've addressed #1769, #1770 and #1771.  At that stage, we should review our requirements and I will get back to AdsAbs if we still require an increase to our limit.

**Comment by GlazerMann** [comment]
I determined that we were doing lots of extras.  Removed them

**Comment by GlazerMann** [comment]
@ms609 They should see a lot fewer requests now.

**Comment by ms609** [comment]
Great! Does this mean we're back under our allocation of daily requests?

**Comment by GlazerMann** [comment]
It looks like we are.

**Comment by GlazerMann** [comment]
Closing since the only other thing we can do is cache them, and that’s another issue
